p2p/proto/state.proto

```diff
@@ -22,15 +22,49 @@ message StateDiff
     repeated ContractDiff contract_diffs = 2;
 }
 
+message EntryPoint {
+    Felt252 selector = 1;
+    Felt252 offset = 2;
+}
+
+message Cairo0Class {
+    bytes abi = 1;
+    repeated EntryPoint externals = 2;
+    repeated EntryPoint l1_handlers = 3;
+    repeated EntryPoint constructors = 4;
+    bytes program = 5;
+}
+
+message SierraEntryPoint {
+    uint64 index = 1;
+    Felt252 selector = 2;
+}
+
+message Cairo1EntryPoints {
+    repeated SierraEntryPoint externals = 1;
+    repeated SierraEntryPoint l1_handlers = 2;
+    repeated SierraEntryPoint constructors = 3;
+}
+
+message Cairo1Class {
+    bytes abi = 1;
+    Felt252 abi_hash = 2;
+    Cairo1EntryPoints entry_points = 3;
+    repeated Felt252 program = 4;
+    Felt252 program_hash = 5;
+    bytes semantic_version = 6;
```

Comment:
What's `semantic_version`? How can it be derived from the feeder gateway [response](https://alpha-mainnet.starknet.io/feeder_gateway/get_class_by_hash?classHash=0x05d92680468ab5aa492f18f623358840e246c9a9462d675722c4ab7b85181ff4)?

Reply:
I don't see `contract_class_version` in the PR now -- are you sure we don't need that?